### PR TITLE
Add docs to flag vkit as pre-beta

### PR DESCRIPTION
**Pull request by michaelbausor**
Added comment at the top of the file and in the class doc block to indicate that vkit is pre-beta.

### CODE REVIEW

**Comment by michaelbausor** [comment]
cc @jdpedrie @dwsupplee

File: src/main/resources/com/google/api/codegen/php/main.snip [hunk 1]
```diff
@@ -51,6 +51,9 @@
    * This file was generated from the file
    * https://github.com/google/googleapis/blob/master/{@xapiClass.protoFilename}
    * and updates to that file get reflected here through a refresh process.
+   *
+   * EXPERIMENTAL: the code generation tool used to create this class has not yet been declared
+   * beta. This class may change more frequently than those which have been declared beta or 1.0.
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/php/main.snip]: maybe also call out potential breaking changes

Comment (michaelbausor) [inline src/main/resources/com/google/api/codegen/php/main.snip]: Updated

**Comment by michaelbausor** [comment]
PTAL

**Comment by dwsupplee** [comment]
LGTM!

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**